Find a functionally equivalent code:android.net.wifi.WifiManager.requestBatchedScan:COMMENT
Method Modifier: public      hidden      
Comment:/**
 * Request a batched scan for access points.  To end your requested batched scan,
 * call stopBatchedScan with the same Settings.
 *
 * If there are mulitple requests for batched scans, the more demanding settings will
 * take precidence.
 *
 * @param requested {@link BatchedScanSettings} the scan settings requested.
 * @return false on known error
 * @hide
 */

@@ -1,6 +1,6 @@
 {
     try {
-        return mService.requestBatchedScan(requested, new Binder());
+        return mService.requestBatchedScan(requested, new Binder(), null);
     } catch (RemoteException e) {
         return false;
     }

